### PR TITLE
renaming blockchain_length and highest_block_length_received

### PR DESCRIPTION
**Pull request by knarz**
The »blockchain length« vs »block height« issue was discussed in discord and the forums.
The daemon already referred to the `blockchain_length` as `block height` in its description.

### CODE REVIEW

File: src/lib/daemon_rpcs/daemon_rpcs.ml [hunk 1]
```diff
@@ -187,9 +187,9 @@ module Types = struct
 
       let num_accounts = int_option_entry "Global number of accounts"
 
-      let blockchain_length = int_option_entry "Block height"
+      let block_height = int_option_entry "Block Height"
```
Comment (cmr) [inline src/lib/daemon_rpcs/daemon_rpcs.ml]: Can you lowercase all the non-starting letters? We've changed style between current testnet and `develop`.

**Comment by cmr** [comment]
@CodaProtocol/core-eng Is this name changed authorized? I think so...

**Comment by bkase** [comment]
@o1pranay cc -- I think we agreed that blockchain_length was misleading. Do you not agree @cmr ?

**Comment by nholland94** [comment]
This introduces a mismatch between the name of the Snarky compatible number module `Length` and the new `block_height` fields in `Consensus_state.t`. If we are going this direction, I think that we should also rename `Length` to `Height`. We may want to look at the transition frontier code as well and make sure there aren't dangling references to length there.

**Comment by mrmr1993** [comment]
This PR is several years out of date. Closing.

**Pull request closed without merging**